modules/openapi-generator/src/main/resources/swift5/libraries/urlsession/URLSessionImplementations.mustache

```diff
@@ -9,6 +9,18 @@ import Foundation
 import MobileCoreServices
 #endif
 
+public protocol NetworkSession {
```

Comment:
@leszek-s and if we make this protocol match the URLSession methods signatures, maybe we can rename it to something like `URLSessionProtocol`.

Reply:
I changed it to URLSessionProtocol as proposed.